### ISSUE TITLE
ask: Setting num_classes =11 or 12 to evaluate the CamVid dataset?

### ISSUE THREAD

**Issue by camhao168** [toimcio/SegNet-tensorflow#1]
Dear toimcio,

Firstly, I really appreciate your work, it helps me a lot :).
I downloaded your pre-trained model of CamVid dataset on  provided google drive link and used it to evaluate the test set. However, the performance metrics are much lower than those on your report table.
I did try to figure out and found that following procedures can produce results equivalent to those of your report:
num_classes = 12  --> after testing, we get a "hist" size of 12x12 --> exclude the 12-th row and column --> get new hist size of 11x11 --> calculate acc, IoU as usual on this new hist confusion matrix.
My question is: is this the procedure that current works are using to evaluate this dataset?
Or it is true that at the beginning, we should set num_classes = 11 to only evaluate on 11 classes as in your report table? If so, in this case, how can we deal with the pixels not labeled in this 11 classes?

Regards
